lightly/transforms/random_crop_and_flip_with_grid.py

```diff
@@ -1,44 +1,30 @@
 import torch
 from torch import nn
-from typing import List
+from typing import List, Tuple, Dict
 from dataclasses import dataclass
 import torchvision.transforms as T
 import torchvision.transforms.functional as F
 from PIL import Image
 
-imagenet_normalize = {"mean": [0.485, 0.456, 0.406], "std": [0.229, 0.224, 0.225]}
 
 @dataclass
 class Location:
     # The row index of the top-left corner of the crop.
-    i: float 
+    top: float 
     # The column index of the top-left corner of the crop.
-    j: float
+    left: float
     # The height of the crop.
-    h: float
+    height: float
     # The width of the crop.
-    w: float
+    width: float
     # The height of the original image.
-    H: float
+    HEIGHT: float
```

Comment:
Please call this `image_height` instead. Having `height` and `HEIGHT` is a bit confusing. Same for width.